powa/static/js/App.vue

```diff
@@ -368,8 +368,7 @@ function loadData() {
 watch(
   () => route.params,
   function (newVal, oldVal) {
-    from.value = route.query.from || defaultFrom;
-    to.value = route.query.to || defaultTo;
+    setFromTo(route.query.from || defaultFrom, route.query.to || defaultTo);
```

Comment:
Maybe we could allow to pass an undefined value, and use defaultFrom only in the service. And then `defaultFrom` wouldn't have to be exposed.